test/test_journalpump.py

```diff
@@ -49,7 +49,8 @@ def test_journalpump_init(tmpdir):  # pylint: disable=too-many-statements
     assert len(a.readers) == 1
     for rn, r in a.readers.items():
         assert rn == "foo"
-        r.create_journald_reader_if_missing()
+        with mock.patch.object(PumpReader, "has_persistent_files", return_value=True):
```

Comment:
I'd prefer patching it in decorator for the test method, but for some reason mock library leaves some temporary files `<MagicMock name=.....>' without properly cleaning up the mocking. Maybe due to multithreading, checked with joins - it does not help. I guess I can leave it like that.